addons/mass_mailing/data/digest_data.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <template id="mass_mailing_main" inherit_id="digest.digest_mail_main">
```

Comment:
As id you can keep the same naming, ``digest_mail_main``. Final xml_id will be prefixed so no collision risk (will be ``mass_mailing.digest_mail_main``. That way when searching for ``digest_mail_main`` in views we find them in Odoo for example.